components/editor/netbeans/Makefile

```diff
@@ -52,8 +57,15 @@ $(BUILD_DIR_64)/.installed: $(BUILD_DIR_64)/.built
 	touch $(BUILD_DIR_64)/.installed;
 
 install: $(BUILD_DIR_64)/.installed
+	$(GSED) -i -e  "1s?.*?$(PYTHON_SHEBANG)?" $(PROTO_DIR)/usr/netbeans/$(COMPONENT_VERSION)/extide/ant/bin/runant.py;
 
+# Runtime dependencies
+REQUIRED_PACKAGES += runtime/java/openjdk21
+REQUIRED_PACKAGES += runtime/perl
```

Comment:
This is already listed below (automatically detected), so please remove the duplicate here.